docs/administration/configuration/configuring-web-ui/index.md

```diff
@@ -0,0 +1,45 @@
+# Configuring web UI
```

Comment:
minor: let's maybe use "Configuring web interface", to avoid using an acronym in the title?

Reply:
Let's also change this in the file name and in the various indexes?